docs/releases.md

```diff
@@ -1,5 +1,32 @@
 # Release Notes
 
+## 15.5.0-preview-20171012-09
+
+### Issue Fixed
+* Fixed Data Collector Attachment issues for legacy TMI test execution workflow.
+* Added error message and help when vstest.console is invoked without arguments.
+* Fixed failure in loading extensions without Identifier Data.
+* Handled Test Host close.
+* TestCase Display Name is displayed instead of FullyQualifiedName.
+* Fixed issues with Static Cover Coverage, Ordered tests through TMI.
+
+### New Features introduced
+* Added Telemetry Infra for Design Mode.
+* Supported running .Net Framework v35 in compat mode.
+* Loclization changes.
+* Automatically find Platform and Framework if not specified explicitly.
+* Changes for making Test Platform as default in Visual Studio.
```

Comment:
Not a feature

Reply:
Making it the default is a not feature per se. Remove that line.